README.md

```diff
@@ -30,9 +30,9 @@ ownCloud is an open-source file sync, share and content collaboration software t
 ## Docker Tags and respective Dockerfile links
 
 - [`latest`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:latest`
+- [`10.11.0-rc.1`](https://github.com/owncloud-docker/server/blob/master/v20.04/Dockerfile.amd64) available as `owncloud/server:10.11.0-rc.1`
```

Comment:
I would like to not put RC's in the Readme and keep the supported stable tags only.

Reply:
ok.